src/plugins/telemetry/public/services/telemetry_notifications/telemetry_notifications.test.ts

```diff
@@ -53,3 +53,27 @@ describe('setOptedInNoticeSeen', () => {
     expect(telemetryService.setUserHasSeenNotice).toBeCalledTimes(1);
   });
 });
+
+describe('shouldShowOptedInNoticeBanner', () => {
+  it('should return true because no banner, not seen and user can change settings', () => {
```

Comment:
```suggestion
  it("should return true because a banner hasn't been shown, the notice hasn't been seen and the user has privileges to edit saved objects", () => {
```